docs/rpc/nostr.md

```
@@ -0,0 +1,231 @@
+
+```bash
+curl --header 'Content-Type: application/json' \
+  --header "Authorization: Nostr <base64-encoded-nip98-event>" \
+  --request POST \
+  --data '{"jsonrpc":"2.0","method":"create_api_key_with_nostr","id":1}' \
+  https://api.btcmap.org/rpc
+```
+
+### Usage Flow
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)